docs/source/documentation.rst

```diff
@@ -0,0 +1,116 @@
+Documentation Guidelines
+-------------------------------------------
+
+Developers of eMach should read this document to understand expectations of the documentation files related to any code contributed to the eMach 
+codebase. This document outlines the requirements for the documentation sections related to ``analyzers`` and ``machine_designs``.
+
+Analyzer Documentation
+++++++++++++++++++++++++++++++++++++++++++++
+
+Each analyzer within the ``eMach`` codebase must be summarized such that someone with a basic understanding of electric machines can understand the
+purpose and structure of the analyzer. The documentation files devoted to each analyzder must contain the following sections:
+
+1. Model Background
+2. User Inputs
+3. User Outputs
+
+Model Background
+*******************************************
+
+The ``model_background`` section needs to provide information to explain the motivation, application, and any other knowledge required to understand
+where and why the analyzer is being applied. All relevant background information for each analyzer must be fully explained in this section. This 
+can be in the form of equations, images, explanations, etc. Any assumptions that are made must be fully explained in this section and any 
+publications that were referenced should be included here as well.
+
+User Inputs
+*******************************************
+
+The ``user_inputs`` section of each analyzer needs to divulge to the user what is required as inputs to the analyzer. This should come in the form of 
+organized tables, example code, and any other information that the user must provide to the analyzer. If additional images are required to understand
+the extent of the analyzer, they should be included here. The tables included in this section must document the input variables and should come in the 
+form of 3 columns. The columns should be listed as the `input variables`, the `units` of the input variables, and a short `description` of each input 
+variable. The example code contained in this section is the code that will run with the ``user_inputs`` and that results in the outputs documented 
+in the following section.
+
+User Outputs
+*******************************************
+
+The ``user_outputs`` section should contain the outputs that will result from running the code documented in the ``user_inputs`` section. This 
+section should contain similar tables, code, and any other post-processing information that will be useful for a proper understanding of the analyzer. 
+Any plots that result from this analyzer should be included in this section as well. The output table should contain the same form as the input table. 
+The code that should be included in this section is any post-processing code needed to calculate the final variables or construct any plots. This code 
+should not be the analyzer code itself, but any code that is used to get from the analyzer outputs to usable data or images. 
```

Comment:
I have a similar comment here. I'd like to see this more concise and clearer on expected content.